MERN/Topic/Frontend/React/Map_in_JSX.md

```diff
@@ -0,0 +1,69 @@
+# **Using .map() in JSX**
+---
+## **What is map?**
+<br>
+
+A Map is a data collection type that holds **key-value pairs** and remembers the original insertion order of the keys. Any value (both objects and primitive values) may be used as either a key or a value. We cannot store a duplicate pair in the map(). It is because of the uniqueness of each stored key. 

```

Comment:
You can provide analogies to other languages. Example: `Similar to dictionary in Python` etc.